### PR TITLE
Fix unit tests

### PR DESCRIPTION
**Pull request by lorenzoPrimi**

### CODE REVIEW

File: ooniprobe/AppDelegate.m [hunk 1]
```diff
@@ -15,12 +15,13 @@ @implementation AppDelegate
 
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+    SharkORM.settings.sqliteJournalingMode = @"DELETE";
+
     if ([self isUITestingEnabled])
         [self copyDBTesting];
         
     [SharkORM setDelegate:self];
     //Default is WAL https://www.sqlite.org/wal.html
```
Comment (bassosimone) [inline ooniprobe/AppDelegate.m:24]: This comment used to apply to the code that you moved; can you also move this comment?

**Review by bassosimone — APPROVED** [review]
🐳 

Thanks for fixing the tests!

**Comment by bassosimone** [comment]
Interestingly enough here we have 50% of failures 😞

**Comment by lorenzoPrimi** [comment]
> Interestingly enough here we have 50% of failures 😞

The failed test is 
```
		-[APITest testDownloadUrls]
````
Because of the endpoint issue

**Comment by bassosimone** [comment]
> Because of the endpoint issue

What endpoint issue? (Maybe let's directly chat about this?)

**Pull request merged**